include/partition/inertial_flow.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef OSRM_PARTITION_INERTIAL_FLOW_HPP_
+#define OSRM_PARTITION_INERTIAL_FLOW_HPP_
+
+#include "partition/graph_view.hpp"
+#include <vector>
+
+namespace osrm
+{
+namespace partition
+{
+
+class InertialFlow
+{
+  public:
+    InertialFlow(const GraphView &view);
+
+    std::vector<bool> ComputePartition(const double balance, const double source_sink_rate);
```

Comment:
The `interface` here is just a placeholder for me to imagine what would happen:

- balance would be a guideline for the cuts to be chosen. Any flow that is computed gives a large number of possible cuts (e.g. `a-b-c-d-e-f-g-h` can be split on any intermediate node with an identical cut size). The balance would guide the flow algorithm which of the possible minimal flows to report / what kind of effort to spend on balancing the cuts in size
- the source/sink-rate would be for the initialisation of the flow algorithm. The inertial flow picks X percent of the nodes as source/sink, right? So that would be the X.
- For the statefulness: we can easily make this not stateful and make the flow algorithm do the `best out of n` part here. That was just to get something working quickly to iterate on.